hbase-client/src/main/java/org/apache/hadoop/hbase/client/ColumnFamilyDescriptorBuilder.java

```diff
@@ -282,6 +283,8 @@
 
   private final static Set<Bytes> RESERVED_KEYWORDS = new HashSet<>();
 
+  private final static Map<Bytes,Function<Bytes,Object>> ATTRIBUTE_VALIDATOR = new HashMap<>();
```

Comment:
If we want to do this same validation server-side, it would be nice to life the map and the validator functions out to their own class which can be more easily re-used.

Reply:
Moved those to separate class.